### ISSUE TITLE
Class timing converting to utc when clicking out of the timing

### ISSUE THREAD

**Issue by Abdulkhan2002** [Doenet/DoenetTools#1547]

**Comment by Abdulkhan2002** [comment]
On the dashboard when choosing class times. Everything works fine with the button controls but once you click out of it, the timing gets converted(goes 5 hours ahead of what you want the time to be scheduled to). It is basically getting converted to UTC time

**Comment by dqnykamp** [comment]
Good catch.  Indeed, it turns out all my due dates are off by 5 hours!  Just enough time to change this before the semester starts.

**Comment by Lyanthropos** [comment]
Is this the same issue as #1555?

**Comment by dqnykamp** [comment]
Oh, I think you are right.  I was thinking this was about Due Dates and Assigned Dates.  I'll make another issue about those.

**Comment by cqnykamp** [comment]
Closing as this pertains to legacy site.
